MIGRATION.md

```diff
@@ -0,0 +1,173 @@
+## Migration Guide
```

Comment:
Are the migration guides meant to start at h2?

Reply:
They all do but... why